constants/chapter.json

```diff
@@ -107,7 +107,7 @@
       {
         "id": 3,
         "title": "A list",
-        "details": ""
+        "details": "I built a function to show cities, then the green button randomly to add a new city at the beginning of the list. Afterwards, I render the country and flag of every city listed above, on the right in blue color, shows the main dish recommended for the first element of the list."
```

Comment:
This could use better grammar. Also the style is not consistent with other descriptions in the page, it should give instructions on how to use it, instead of explaining what you did

Reply:
Refactored